src/Mod/Path/PathScripts/PathOpTools.py

```diff
@@ -291,7 +291,6 @@ def offsetWire(wire, base, offset, forward, Side=None):
             return Part.Wire([edge])
 
         # if we get to this point the assumption is that makeOffset2D can deal with the edge
```

Comment:
Then this comment should be removed too, right?

Reply:
Not necessarily. The comment signals to the dev useful information